From 6c0014b397106a082fa5bc89603e132acb3c052a Mon Sep 17 00:00:00 2001
From: Octavian Purdila <octavian.purdila@nxp.com>
Date: Thu, 4 May 2017 15:58:24 +0300
Subject: [PATCH 1755/5242] MLK-14786: ARM: dts: imx6x-sabreauto-m4: add reg
 info to the rpmsg node

commit  1d514fa3995b6e8ddb047a7ee24bb8b815e20ec0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Without this information probing of imx-rpmsg fails as it is not able
to setup the vring due to missing allocated physical memory.

Signed-off-by: Octavian Purdila <octavian.purdila@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto-m4.dts |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto-m4.dts b/arch/arm/boot/dts/imx6sx-sabreauto-m4.dts
index 51ac31c..a168992 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto-m4.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto-m4.dts
@@ -58,6 +58,8 @@
 };
 
 &rpmsg{
+	vdev-nums = <1>;
+	reg = <0xbfff0000 0x10000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

